.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,64 @@
+name: 🐛 Bug report
+description: Report a bug or unexpected behavior
+title: "Bug title"
+labels: []
+body:
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe the bug
+      description: Please enter a clear and short description of your bug here.
+      placeholder: A crash occurs if X is called from Y.
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Operating system version
+      description: On what operating system did this bug occur? Provide information of the affected OS and its version here.
+      placeholder: |
+        Windows 11 (Build 10.0.22463.1000.rs_prerelease.210917-1503), Ubuntu 20.04 LTS
+  - type: textarea
```

Comment:
What do you think about having a drop down that contains all the (pre)release branches and `main` (default to main)?

Reply:
Sounds good, I'll work on that.